src/components/Form.js

```diff
@@ -0,0 +1,38 @@
+import styles from "@/styles/Form.module.scss"
+
+function Form(props) {
+   
```

Comment:
Make a name of file and component ContactForm

Reply:
Fixed.